woocommerce/wc-admin
# Fix pre-release script

After [making FeaturePlugin class internal](https://github.com/woocommerce/woocommerce-admin/pull/8317), the `FeaturePlugin.php` path changed. This PR fixes the `FeaturePlugin.php` path in the `pre-release` script. 

(No changelog)
### Detailed test instructions:

- Check out this branch.
- Run `npm run pre-release` in your terminal.
- Base the release on `fix/pre-release-script`.
- You'll see the question: `What version would you like to release?`. Answer: `my-testing-release`.
- Verify that line 158 in `src-internal/Admin/FeaturePlugin.php` was updated correctly.
